packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -206,7 +206,7 @@ export function ArticleMetadataPanel({value, infoData, onClose, onChange}: Artic
             <div style={{paddingBottom: '20px'}}>{t("articleEditor.panels.totalCharCount",{totalCharCount: infoData.charCount})}</div>
               <FormGroup>
                 <ControlLabel>{t('articleEditor.panels.preTitle')}
-                <label style={{float: 'right'}}> {`${value.preTitle.length}`} </label>
+                <label style={{float: 'right'}}> {`${value.preTitle.length}`}/{preTitleMax} </label>
```

Comment:
@HanaAyman Correct me if I am wrong, I guess you are having the length label in different styles, so this one is different than on line 140 and also 164 and 221